jwt.js

```diff
@@ -140,7 +141,8 @@ function fastifyJwt (fastify, options, next) {
       options = Object.assign({}, decodeOptions)
     }
 
-    return jwt.decode(token, options)
+    const decoder = createDecoder(options)
```

Comment:
We should not be creating a decoder every time. It's expensive and remove most of the benefits of fast-jwt.